machines/pod_1x1/Makefile.machine.include

```diff
@@ -0,0 +1,31 @@
+BSG_MACHINE_PODS_X									  = 1
+BSG_MACHINE_PODS_Y									  = 1
+
+BSG_MACHINE_GLOBAL_X                  = 16
+BSG_MACHINE_GLOBAL_Y                  = 8
+
```

Comment:
Are GLOBAL_X/GLOBAL_Y the pod dimensions?

Is there a reason GLOBAL is still used? I know historically it was used to indicate what the GLOBAL eva could address?

If not - can we change it to something like BSG_MACHINE_POD_DIM_X?